ui/FilamentPanel.go

```diff
@@ -109,17 +111,50 @@ func (this *filamentPanel) initialize() {
 
 
 	// Third row
+	column := 0
 	this.temperatureButton = utils.MustButtonImageStyle("Temperature", "heat-up.svg", "color1", this.showTemperaturePanel)
-	this.Grid().Attach(this.temperatureButton, 0, 2, 1, 1)
+	this.Grid().Attach(this.temperatureButton, column, 2, 1, 1)
+	column++
 
 	// The select tool step button is needed by some of the other controls (to get the name/ID of the tool
 	// to send the command to), but only display it if multiple extruders are present.
 	extruderCount := utils.GetExtruderCount(this.UI.Client)
 	if extruderCount > 1 {
-		this.Grid().Attach(this.selectExtruderStepButton, 1, 2, 1, 1)
+		this.Grid().Attach(this.selectExtruderStepButton, column, 2, 1, 1)
+		column++
+	}
+
+	if utils.FilamentManagerPluginIsInstalled(this.UI.Client) {
```

Comment:
If octoprint isn't running when the filament panel is initialized, we won't add the filament manager button unless we restart Octoscreen.